test Fixture with speci@l chars/camel-maven-quarkus-project/.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "git.openRepositoryInParentFolders": "never",
+  "redhat.telemetry.enabled": false,
+  "workbench.secondarySideBar.defaultVisibility": "hidden",
+  "window.zoomLevel": -1.5
+}
```

Comment:
Is this meant to be committed? Perhaps setting the workbench to make the tests easier?